test/browser/fixtures/locators/blog.test.tsx

```diff
@@ -18,7 +18,6 @@ test('renders blog posts', async () => {
 
   await expect.element(secondPost.getByRole('heading')).toHaveTextContent('qui est esse')
 
-  // TODO: click doesn't work on webdriverio when iframe is scaled
```

Comment:
Technically the scaling issue still exists for with headed UI. This test still fails locally with:

```js
PROVIDER=webdriverio BROWSER=chrome pnpm -C test/browser test-locators --browser.headless=false
```